### ISSUE TITLE
Errors thrown in wrappers and WasmWrapper.ts are thrown instead of returned as errors in InvokeResult

### ISSUE THREAD

**Issue by krisbitney** [Web3-API/prototype#1311]
**Describe the bug**
When Wasm wrappers encounter errors, the errors are not returned in `InvokeResult`. Instead, they are thrown. I'm not sure if this is intentional.

**Expected behavior**
I assumed these errors would be returned in `InvokeResult`

**Screenshots / logs**
<img width="291" alt="Screen Shot 2022-10-10 at 7 01 46 PM" src="https://user-images.githubusercontent.com/39842820/194884017-42c202eb-9370-4a28-9d89-5c9b07177aa3.png">

<img width="799" alt="Screen Shot 2022-10-10 at 6 58 02 PM" src="https://user-images.githubusercontent.com/39842820/194883513-444980c4-829e-4540-8962-56cd21a1f37d.png">

**Relevant environment details:**
- OS: MacOS with M1
- Rust Wasm wrapper
- Jest test suite

**Comment by cbrzn** [comment]
I think these are related https://github.com/polywrap/toolchain/issues/701 and would like to work on a game plan for this - @krisbitney are you down to jam about it this week?

**Comment by krisbitney** [comment]
> I think these are related #701 and would like to work on a game plan for this - @krisbitney are you down to jam about it this week?

Yeah, sounds good! I have a lot of time open today and tomorrow. Is there any particular time or day that works well for you, or should we just try to catch each other online?

**Comment by cbrzn** [comment]
will dm you :-D @krisbitney

**Comment by krisbitney** [comment]
Is this still happening? I haven't seen it in a long time
